### PR TITLE
[SDEV-378] - spruce-skill has big search examples

### PR DESCRIPTION
**Pull request by liquidg3**
## What does this PR do?

Brings in big search examples

## Type

- [x] Feature
- [ ] Bug
- [ ] Tech debt

## What are the relevant tickets?

- [SDEV3-378](https://sprucelabsai.atlassian.net/browse/SDEV3-378)

## Does this add new dependencies? Is there an installation procedure? (yarn, bower, npm, etc)

NA

## Screenshots (if appropriate)

NA

## What gif best describes this PR or how it makes you feel?

NA

### CODE REVIEW

File: packages/spruce-skill/server/events/big-search.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+						}
+					}
+				})
+			)
+		}
+
+		sections.push(section1)
+
+		// or you can search any source you want and mark them as needing to be imported
+		const dummyResults: IBigSearchResult[] = []
```
Comment (gcpantazis) [inline packages/spruce-skill/server/events/big-search.js]: Probably should improve this to demonstrate pagination, search.

File: packages/spruce-skill/server/events/big-search.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+			event: {
+				payload: { limit, offset, search, testing }
+			}
+		} = ctx
+
+		// each section
+		const sections: IBigSearchSection[] = []
+
+		// do whatever you want with these
+		console.log('ignoring', search, testing)
```
[the remark this answers is not in the document]
Comment (liquidg3) [reply packages/spruce-skill/server/events/big-search.js]: added those to suppress unused vars errors (but still wanted to show the full API)

File: packages/spruce-skill/server/events/big-search.js (hunk 1 above)
Comment (liquidg3) [reply packages/spruce-skill/server/events/big-search.js]: We have paging in the query above (lines 33/34), just not in this dummy section. Given i don't have an actual data source to pull from, i'm not quite sure how i'd show paging without baking a lot out.

File: packages/spruce-skill/server/events/import-from-big-search.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+					routeParams: {
+						organizationId: organization.id,
+						locationId: location.id,
+						userId: updatedUser.id
+					}
+				}
+			}
+		} else {
+			// Step 2.1. If they did not pass a match id, lets make sure this user doesn't already exist (searching by phone is the most reliable way)
+			// you can search by anything you want and
```
Comment (kengoldfarb) [inline packages/spruce-skill/server/events/import-from-big-search.js]: and...?

Comment (liquidg3) [reply packages/spruce-skill/server/events/import-from-big-search.js]: haha, fixing

**Pull request merged**